ts/components/session/LeftPaneSettingSection.tsx

```diff
@@ -212,6 +212,11 @@ export class LeftPaneSettingSection extends React.Component<Props, State> {
         title: window.i18n('privacySettingsTitle'),
         hidden: false,
       },
+      {
+        id: SessionSettingCategory.Blocked,
+        title: window.i18n('settingsUnblockHeader'),
```

Comment:
Could we make this `blockedSettingsTitle` just to fit in with the other names?

Reply:
Of course, we can :)